GOVERNANCE.md

```diff
@@ -22,30 +18,226 @@ New maintainers must be nominated by an existing maintainer and must be elected
+to a future proposal, that might be mentioned here as well.
+
+#### API Changes
+Describe how Pinniped's API will change. APIs include CLI commands,
+HTTP endpoints, aggregated API endpoints, CRDs, etc.
+Detail changes to their inputs, outputs, and behavior.
+What will the default values be for any new fields or parameters?
+
+#### Upgrades
+Describe how upgrading to a new version of Pinniped which includes
+these features would work. Are the new changes backwards compatible?
+Can new and old versions of the CLI and servers be mixed?
+Will it be possible to downgrade after upgrading?
+
+#### Tests
+What kind of integration tests could be used to test the new features?
+
+#### New Dependencies
+Would any significant new project dependencies be needed to support
+the implementation? Consider Golang libraries, CI infrastructure, etc.
+
+#### Performance Considerations
+Any concerns with scalability, performance, or reliability for the
+implementation?
+
+#### Observability Considerations
+Any new log statements or other considerations to make this feature
+observable and debuggable for admin users?
+
+#### Security Considerations
+How does the proposal consider security? What makes the new features
+secure?
+
+#### Usability Considerations
+How does the proposal consider usability for the end user (kubectl user)
+and for the admin user who installs and configures Pinniped?
+
+#### Documentation Considerations
+How will users discover the new features? Will docs changes be required
+during implementation?
+
+### Other Approaches Considered
+Mention of other reasonable ways that the problem(s)
+could be addressed with rationale for why they were less
+desirable than the proposed approach.
+
+## Open Questions
+A list of questions that need to be answered.
+
+## Answered Questions
+A list of questions that have been answered.
+
+## Implementation Plan
+Who will implement this proposal once it is finished and approved?
+Do you already have ideas for how you might approach the implementation
+in an iterative fashion? For a large proposal with an iterative plan,
+where might you draw the line to define a minimum viable version?
+
+## Implementation PRs
+This section is a placeholder to list the PRs that implement this proposal.
+This section should be left empty until after the proposal is approved.
+After implementation, the proposal can be updated to list related
+implementation PRs.
+```
+
+## Proposal States
+| Status | Definition |
+| --- | --- |
+| `draft` | The proposal is actively being written by the proposer. Not yet ready for review. |
+| `in-review` | The proposal is being reviewed by the community and the project maintainers. |
+| `accepted` | The proposal has been accepted by the project maintainers. |
+| `rejected` | The proposal has been rejected by the project maintainers. |
+| `implemented` | The proposal was accepted and has since been implemented. |
+
+## Lifecycle of a Proposal
```

Comment:
I would like to see https://github.com/vmware-tanzu/pinniped/pull/973#discussion_r802926977 addressed here.